### DISCUSSION TITLE
a simple  flet icons application

### DISCUSSION THREAD

**Discussion by modaye** [flet-dev/flet#338] [Show and tell]
I built a flip icon app. Do you have any suggestions

![](https://github.com/modaye/Flet-Icon/blob/master/images/img.png)

[传送门](https://github.com/modaye/Flet-Icon)

**Comment by FeodorFitsner** [comment]
Cool! Host it on either Replit, Fly.io or Heroku (which is still free), so we could play with it only and leave our feedback :)

**Reply by modaye** [reply]
https://fleticon.modaye.repl.co   it in  Replit  But it lost the clipboard function

**Reply by FeodorFitsner** [reply]
Looks great! Could you please describe in a few words what does the app do?

**Reply by modaye** [reply]
Of course, it is an icon application that provides fast icon search and actual appearance display for flet users,

1. Query through the sidebar and search box (regular expressions are supported)

2. Click icon in desktop program to automatically copy icon code to clipboard

**Reply by FeodorFitsner** [reply]
Nice!

**Reply by ndonkoHenri** [reply]
> https://fleticon.modaye.repl.co   it in  Replit  But it lost the clipboard function

Can I have the source code to this please.

**Reply by modaye** [reply]
you get https://github.com/modaye/Flet-Icon

**Reply by ndonkoHenri** [reply]
Thanks mate.
